packages/react-components/src/components/Stepper/Stepper.spec.tsx

```diff
@@ -0,0 +1,80 @@
+import { render, screen } from '@testing-library/react';
+
+import { Stepper } from './Stepper';
+
+describe('Stepper', () => {
+  it('displays correct number of steps', () => {
+    render(<Stepper activeStep={2} steps={4} />);
+
+    const stepsContainer = screen.getByTestId('stepper-container');
+
+    expect(stepsContainer?.children.length).toBe(4);
+  });
+
+  it('marks steps as completed and active correctly', () => {
+    render(<Stepper activeStep={2} steps={3} />);
+
+    const steps = screen.getByTestId('stepper-container').children;
+
+    expect(steps[0].className).toContain('completed');
+    expect(steps[1].className).toContain('completed');
+    expect(steps[2].className).toContain('active');
```

Comment:
In my opinion we should avoid checking the component state through classes, because in various cases they are the most common element of the component that is modified.

`aria` labels are ideal for this type of thing, steps could have `aria-label` or `role` and also the attribute `aria-completed` or others, which could be referred to in tests and would be consistent with the internal logic of the component.
WDYT? @JoannaSikora

Reply:
on it 🫡